test/stability/testSingleWorkingFF/testSingleWorkingFF.go

```diff
@@ -274,14 +272,13 @@ func setParameters(testScenario uint) (err error) {
 		}
 		// Test expects to receive 20% of packets on 0 port and 80% on 1 port
 		lessPercent = 20
-		eps = 4
 	case partition, vpartition:
 		// Test expects to receive 10% of packets on 0 port and 90% on 1 port
 		lessPercent = 10
 	case pcopy:
 		// Test expects to receive 50% of packets on 0 port and 50% on 1 port (200% total)
 		lessPercent = 50
-		passedLimit = 150 // ~85 * 2
+		passedLimit = 196 // ~98 * 2
 	case handle, vhandle:
```

Comment:
Can you use a named constant defined at the beginning of the file instead of magic number 98?

Reply:
It is just name inside setParameters